redfish/provider/provider.go

```diff
@@ -99,6 +99,8 @@ func (*redfishProvider) Resources(_ context.Context) []func() resource.Resource
 	return []func() resource.Resource{
 		NewPowerResource,
 		NewUserAccountResource,
+		NewSimpleUpdateResource,
+		NewDellIdracAttributesResource,
```

Comment:
Is this entry correct?

Reply:
I mean, "NewDellIdracAttributesResource"?

Reply:
yes, Idrac attributes resource migrated to tpf in previous pr.